components/expenses/PayoutBankInformationForm.js

```diff
@@ -362,6 +362,15 @@ const PayoutBankInformationForm = ({ isNew, getFieldName, host, fixedCurrency, i
   // Display spinner if loading
   if (loading) {
     return <StyledSpinner />;
+  } else if (host.slug === TW_API_COLLECTIVE_SLUG && !data?.host?.transferwise) {
```

Comment:
I think testing for `host.transferwise` should be enough due to the line `host = data?.host || host`.
```suggestion
  } else if (!host?.transferwise) {
```

Reply:
Thanks, I've simplified the condition as you suggested. Seems to work fine. 👍🏽